shellwhat/checks/has_funcs.py

```diff
@@ -185,10 +181,13 @@ def has_expr(
 
     # do the comparison
     if (strict and res != stu_output) or (res not in stu_output):
-        _msg = state.build_message(
-            incorrect_msg, fmt_kwargs={"expr": expr, "output": output}
-        )
-        state.report(_msg)
+        if isinstance(incorrect_msg, FeedbackComponent):
```

Comment:
Is this needed if the sct author passes in a FeedbackComponent? It doesn't look like the wrappers do?

Reply:
It's used in has_cwd, which delegates to (a partial wrapper of) has_expr.